docker/ci/Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN /cicd/scripts/release/install_release_script_dependencies.R
 # Copy rest from of scripts
 COPY armadillo/scripts/release/* /cicd/scripts/release/
 
+COPY armadillo/scripts/release/xenon-survival.R /cicd/scripts/release/
+
```

Comment:
this should go

The error is in `docker/bin/prepare.bash`

But we need to put all extracted scripts into a folder say `lib` or `src` then include them from there

Reply:
Extracted from `release-test.R` I meant